ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -473,6 +474,37 @@ def test_paddle_atanh(
     )
 
 
+# broadcast_shape
+@handle_frontend_test(
+    fn_tree="paddle.broadcast_shape",
+    shapes=nph.mutually_broadcastable_shapes(
```

Comment:
use `helpers.mutually_broadcastable_shapes` strategy instead as it contains additional constraints over the strategy you used